src/commands/distribute/release.ts

```diff
@@ -81,17 +86,31 @@ export default class ReleaseBinaryCommand extends AppCommand {
     debug("Extracting release ID from the release URL");
     const releaseId = this.extractReleaseId(releaseUrl);
 
-    debug("Distributing the release");
-    await this.distributeRelease(client, app, releaseId, releaseNotesString);
+    debug("Setting release notes");
+    await this.putReleaseDetails(client, app, releaseId, releaseNotesString);
+
+    if (!_.isNil(this.distributionGroup)) {
+      debug("Distributing the release to a group");
+      await this.distributeRelease(client, app, releaseId);
```

Comment:
We can now have a situation where publishing to a distribution group succeeded, followed by a failure to publish to the store, or the other way around cause things are async, in which case you would only see the failure and not get confirmation of the success.

Reply:
The only thing that can happen is successful distribution to a group but failure at the store release.